go.mod

```diff
@@ -4,10 +4,35 @@ go 1.12
 
 require (
```

Comment:
This block compiles and avoids the replace statements:

```
require (
	github.com/go-logr/logr v0.1.0
	github.com/google/gofuzz v1.0.0 // indirect
	github.com/hashicorp/golang-lru v0.5.0 // indirect
	github.com/json-iterator/go v1.1.6 // indirect
	github.com/modern-go/reflect2 v1.0.1 // indirect
	github.com/onsi/ginkgo v1.8.0
	github.com/onsi/gomega v1.5.0
	golang.org/x/net v0.0.0-20190613194153-d28f0bde5980
	golang.org/x/sys v0.0.0-20190616124812-15dcb6c0061f // indirect
	golang.org/x/text v0.3.2 // indirect
	gopkg.in/yaml.v2 v2.2.2 // indirect
	k8s.io/apimachinery v0.0.0-20190704094733-8f6ac2502e51
	k8s.io/client-go v11.0.1-0.20190704100234-640d9f240853+incompatible
	k8s.io/cluster-bootstrap v0.0.0-20190703212826-5ad085674a4f // indirect
	k8s.io/component-base v0.0.0-20190703210340-65d72cfeb85d // indirect
	k8s.io/kube-openapi v0.0.0-20190228160746-b3a7cee44a30 // indirect
	k8s.io/kubernetes v1.14.3
	sigs.k8s.io/controller-runtime v0.2.0-beta.4
)
```

To generate this block I deleted most of the imports, I kept kubernetes at v1.14.3; used apimachinery at release-1.14 and client-go at release-11.0.

Reply:
thanks!

Reply:
If you force-vendor `cluster-api` in hack/tools.go like we do upstream, you should get the same versions we have in CAPI for these dependencies

Reply:
does that require we build from vendor?

Reply:
Not really, it'll just get the modules. I assume later down the line, you'll need to vendor it anyways

Reply:
ohh nvm, i get it

Reply:
Dope. Let's add that later on down the line!

Reply:
@vincepri you cool if we leave it like this for now and get the hack stuff in when we need the vendor dir? I recognize we will need it sooner rather than later, but i think this change is fine as is